vitrina/identifiers/migrations/0004_auto_20250821_1358.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.25 on 2025-08-19 10:58
+
+from django.db import migrations, models
+
+
+def set_risr_validation_options(apps, schema_editor):
+    Agency = apps.get_model('vitrina_identifiers', 'Agency')
+    try:
+        agency = Agency.objects.get(code="risr")
+        agency.identifier_validation_type = "REGEXP"
+        agency.identifier_validation_options = r"^\d{4}$"
+        agency.save()
+    except Agency.DoesNotExist:
+        pass
```

Comment:
You can do it one line:
```suggestion
    Agency.objects.filter(code="risr").update(
        identifier_validation_type="REGEXP",
        identifier_validation_options=r"^\d{4}$"
    )
```

Reply:
Added